packages/rancher-alerting-drivers/charts/values.yaml

```diff
@@ -6,6 +6,9 @@ global:
   cattle:
     # the registry where all images will be pulled from
     systemDefaultRegistry: ""
+  kubectl:
+    repository: rancher/kubectl
+    tag: v1.20.2
```

Comment:
According to the offline discussion, we will use the `rancher/kubectl` image to avoid the requirements for root user permission.